### ISSUE TITLE
local variable 'has_datapacks' referenced before assignment

### ISSUE THREAD

**Issue by Zaczero** [cdbbnnyCode/modpack-installer#26]
```sh
python install.py foo.zip --manual
```

```log
Building Processors
Injecting profile
Finished!
Copying overrides
mods-resourcepacks...
mods...
resources...
scripts...
config...
Done!



To launch your new modpack, just open the Minecraft launcher normally.
The modpack will be available in your installations list.
Traceback (most recent call last):
  File "/home/user/Source/modpack-installer/install.py", line 260, in <module>
    main(args.zipfile, args.mcdir, args.forge_disable)
  File "/home/user/Source/modpack-installer/install.py", line 247, in main
    if has_datapacks:
UnboundLocalError: local variable 'has_datapacks' referenced before assignment
```

**Comment by cdbbnnyCode** [comment]
Oops! Thanks for reporting.

**Comment by cdbbnnyCode** [comment]
Fixed in [dd83180](https://github.com/cdbbnnyCode/modpack-installer/commit/dd8318026010fe73cecf2123dc48b294c9108c30)
